Makefile.toml

```diff
@@ -91,6 +97,8 @@ command = "cross"
 args = [
   "build",
   "--target",
+  "--features",
+  "${FEATURES}",
```

Comment:
these two lines are splitting `--target` and `${CROSS_TARGET}` which belong together